docs/_glossary/HelpWindow.md

```diff
@@ -5,4 +5,4 @@ show-in: [ug, dg]
 
 A pop-up window containing help information, shown only after calling a `help` command.
 
-To view more information, refer to the [Layout](/UserGuide.html#layout) section of the User Guide.
+To view more information, refer to the [Layout](#layout) section of the User Guide.
```

Comment:
Ditto

Reply:
Yup, this needs to be fixed, but this is not the fix. The issue is that if we don't use a fragment, this will break the PDF version of the UG since it will try to open a website instead. Working on a fix in the validation.